src/main/kotlin/com/cognifide/gradle/aem/pkg/tasks/PackageCompose.kt

```diff
@@ -141,6 +141,7 @@ open class PackageCompose : ZipTask() {
         description = "Composes CRX package from JCR content and built OSGi bundles"
 
         archiveBaseName.set(aem.baseName)
+        destinationDirectory.set(AemTask.temporaryDir(aem.project, name))
```

Comment:
no longer CRX packages in `distributions` but in dedicated `build/aem/packageCompose` dir; breaking change will be mentioned in release notes. most important in this release!